CRM/Utils/Money.php

```diff
@@ -44,12 +44,15 @@ class CRM_Utils_Money {
    * @param bool $onlyNumber
    * @param string $valueFormat
    *   The desired monetary value display format (e.g. '%!i').
+   * @param bool $onlyNumberReplaceSeparators
```

Comment:
Dang - this just highlights to me that we should have a separate function rather than this big overload - don't we have one?

Reply:
Nope we don't because the replacement only occurs in this function and only if we have not specified to return just the number

Reply:
@seamuslee001 I gave this a quick test - https://github.com/civicrm/civicrm-core/pull/18151